src/maintainer/infrastructure.rst

```diff
@@ -285,8 +285,14 @@ here `<https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/
 GitHub Actions
 --------------
 
-We use GitHub actions to rerender feedstocks and also run our pull request automerge service. We do not currently support builds on
-GitHub Actions.
+We use GitHub actions to rerender feedstocks and also run our pull request automerge service.
+We also support builds on GitHub Actions.
```

Comment:
This is not entirely true. There's support for GitHub Actions workflows on conda-forge as long as they do not use Github-hosted runners. However, users can use conda-smithy to populate a feedstock (outside conda-forge!) that does use Github Actions with Github-hosted runners. The reasoning is that conda-forge is too big for GH-hosted runners on our org, and we would easily consume all the runners that are used to run other infrastructure services.